docs/settings/apm-settings.asciidoc

```diff
@@ -77,6 +77,9 @@ Maximum number of child items displayed when viewing trace details. Defaults to
 `xpack.observability.annotations.index` {ess-icon}::
 Index name where Observability annotations are stored. Defaults to `observability-annotations`.
 
+`xpack.apm.searchAggregatedServiceMetrics` {ess-icon}::
```

Comment:
Is there an issue to add this to the cloud allow list?

Reply:
Maybe controversial but perhaps we don't want to include `xpack.apm.searchAggregatedServiceMetrics` at all? 

Assume `auto` similar to `xpack.apm.searchAggregatedTransactions`. 

For now this is still optin from APM server too but as soon as were confident do we ever want users to disable this?

Reply:
With `auto` on the service inventory page we will have to send 2 additional (but quite fast) requests _(main stats, detailed stats)_ to determine if it has aggregated services. 

it will effect all the users while it will be disabled on the APM server for the majority. 

Once we are confident I expect it will work as `xpack.apm.searchAggregatedTransactions`